pkg/api/v1beta3/dynakube/dynakube_status.go

```diff
@@ -162,6 +162,10 @@ type EnrichmentRule struct {
 }
 
 func (rule EnrichmentRule) ToAnnotationKey() string {
+	if rule.Mapping == "" {
```

Comment:
Why do you have to check here, if you already skipped the item in the for loop?

Reply:
Mainly for consistency, has no use for now. Also kinda make sense to not append "nothing" to a prefix 😅 

Was thinking of doing a check like `if rule.ToAnnotationKey() == "" { continue }`, but I found the  `if rule.Mapping == "" { continue }` more descriptive.